manifests/init.pp

```diff
@@ -146,6 +146,14 @@
       }
     }
 
+    file {'/etc/httpd/conf.d/ssl.conf':
+      ensure  => file,
+      content => '# Managed by Puppet',
```

Comment:
I think it could be worth putting in here either the reason for this or a link to either the puppet-apache issue or our issue to help anyone in the future who asks why this is the way it is to know.